charts/temporal/tests/web_service_test.yaml

```diff
@@ -0,0 +1,37 @@
+suite: test server service
+templates:
+  - web-service.yaml
+tests:
+  - it: includes the provided clusterIP to make the service headless
+    template: templates/web-service.yaml
+    documentSelector:
+      path: kind
+      value: Service
+      matchMany: true
+    values:
+      - ../values.yaml
```

Comment:
You don't need to set `values`, it will use the default file.

Reply:
Done.